src/styles/main.scss

```diff
@@ -1,3 +1,28 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+@import "./variable.scss";
+@import "./blocks/header.scss";
+@import "./blocks/card.scss";
+@import "./blocks/page.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/product.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+html,
+body,
+div,
+h1,
+h2,
+h3,
+h4,
+p,
+span,
+a,
+ul,
+ol,
+li {
   margin: 0;
+  padding: 0;
 }
```

Comment:
move to the reset file
also, a lot of tags don't have padding, so, filter these tags